data/json/itemgroups/Locations_MapExtras/locations_commercial.json

```diff
@@ -1,1928 +1,1376 @@
-      { "group": "writing_utensils", "prob": 40 }
-    ]
-  },
-  {
-    "id": "dollar_kitchen",
+    "id": "chair_torso",
     "type": "item_group",
-    "items": [
-      [ "cup_plastic", 20 ],
-      [ "rolling_pin", 20 ],
-      [ "bowl_plastic", 20 ],
-      [ "canteen", 10 ],
-      [ "flask_hip", 15 ],
-      [ "fork", 25 ],
-      [ "sponge", 30 ],
-      [ "spoon", 25 ],
-      [ "knife_butter", 25 ],
-      [ "pan", 15 ],
-      [ "pot", 15 ],
-      [ "teapot", 15 ],
-      { "group": "tinware", "prob": 8 }
-    ]
+    "subtype": "distribution",
+    "entries": [ { "item": "suit", "prob": 70 }, { "item": "waistcoat", "prob": 70 }, { "item": "dress_shirt", "prob": 70 } ]
   },
   {
-    "id": "dollar_food",
+    "id": "chair_pants",
     "type": "item_group",
-    "items": [
-      { "group": "candy_chocolate", "prob": 45 },
-      { "item": "licorice", "prob": 15 },
-      { "group": "salty_snacks", "prob": 80 },
-      [ "peanut_unshelled", 20 ],
-      [ "almond_unshelled", 20 ],
-      [ "cereal", 25 ],
-      [ "cereal3", 25 ],
-      [ "soup_tomato", 25 ],
-      [ "soup_chicken", 25 ],
-      [ "lasagne_raw", 25 ],
-      [ "macaroni_raw", 25 ],
-      [ "can_beans", 20 ],
-      [ "can_spam", 20 ],
-      [ "can_pineapple", 20 ],
-      [ "can_tuna", 20 ],
-      [ "can_cheese", 20 ],
-      [ "dry_rice", 20 ],
-      [ "dry_beans", 20 ],
-      [ "dry_lentils", 50 ],
-      [ "cookies", 25 ],
-      [ "bread", 40 ]
-    ]
-  },
-  {
-    "type": "item_group",
-    "id": "coffee_shop",
-    "items": [
-      { "group": "candy_chocolate", "prob": 530 },
-      { "item": "licorice", "prob": 15 },
-      { "group": "softdrinks_canned", "prob": 470 },
-      { "group": "snacks", "prob": 560 },
-      [ "coffee_raw", 70 ],
-      [ "water_clean", 90 ],
-      [ "water_mineral", 10 ],
-      [ "almond_milk", 30 ],
-      [ "soy_milk", 30 ],
-      [ "juice", 20 ],
-      [ "sandwich_t", 30 ],
-      [ "sandwich_pb", 30 ],
-      [ "sandwich_pbj", 30 ],
-      [ "sandwich_pbh", 30 ],
-      [ "sandwich_pbm", 30 ],
-      [ "pizza_veggy", 8 ],
-      [ "pizza_cheese", 8 ],
-      [ "pizza_meat", 8 ],
-      [ "hamburger", 40 ],
-      [ "fish_fried", 8 ],
-      [ "fish_sandwich", 8 ],
-      [ "cheeseburger", 40 ],
-      [ "sloppyjoe", 8 ],
-      [ "hotdogs_frozen", 12 ],
-      [ "hotdogs_cooked", 35 ],
-      [ "corndogs_frozen", 30 ],
-      [ "blt", 40 ],
-      [ "salted_fish", 15 ],
-      [ "pie", 20 ],
-      [ "pie_meat", 20 ],
-      [ "pie_veggy", 20 ],
-      [ "pie_maple", 10 ],
-      [ "cinnamon", 15 ],
-      [ "coffee_bean", 20 ],
-      [ "roasted_coffee_bean", 50 ],
-      [ "restaurantmap", 5 ],
-      [ "touristmap", 2 ],
-      [ "brioche", 50 ]
-    ]
+    "subtype": "distribution",
+    "entries": [ { "item": "pants", "prob": 70 } ]
   },
   {
+    "id": "chair_feet",
     "type": "item_group",
-    "id": "pawn",
```

Comment:
This means you deleted some itemgroups (or just the ids) in this file, not sure how many.
The solution that comes to mind and isn't super complicated would be restoring the file to before your changes and do your changes again.
![image](https://user-images.githubusercontent.com/58050969/114903713-b03bb100-9ddc-11eb-9d71-333c33380627.png)

Reply:
Found out about that, fixing it soon

Reply:
fix pushed